lib/AST/ASTContext.cpp

```diff
@@ -871,7 +871,8 @@ ClassDecl *ASTContext::get##NAME##Decl() const { \
       /* Note: use unqualified lookup so we find NSError regardless of */ \
       /* whether it's defined in the Foundation module or the Clang */ \
       /* Foundation module it imports. */ \
```

Comment:
This comment is wrong to begin with; a qualified lookup will look into re-exported modules as well.

Reply:
Yeah, and this could probably be cleaned up to just call lookupInModule().

Reply:
No, that's the one that *doesn't* look in re-exported modules.

Reply:
Can you double check so I'm not missing something? I thought lookupValue() is the one that only looks in the module itself whereas lookupInModule() does all the extra work of walking the graph...?